packages/editable/src/use-editable.ts

```diff
@@ -212,6 +212,7 @@ export function useEditable(props: UseEditableProps = {}) {
         ref: mergeRefs(ref, previewRef),
         children: isValueEmpty ? placeholder : value,
         hidden: isEditing,
+        w: "full",
```

Comment:
Thanks for your work here @anubra266.

I usually consider hooks like this to be agnostic of Chakra UI styling. In this case, I'll move the `w=full` to the `EditablePreview` component and pass it as a prop there.